programs/merkle-distributor/src/instructions/claim/new_claim.rs

```diff
@@ -117,10 +122,15 @@ pub fn handle_new_claim(
 
     let node = hashv(&[LEAF_PREFIX, &node.to_bytes()]);
 
-    require!(
-        verify(proof, distributor.root, node.to_bytes()),
-        ErrorCode::InvalidProof
+    let verify_res = verify_partial_merkle(
+        partial_merkle_tree.root,
+        node.to_bytes(),
+        partial_merkle_tree.depth,
+        initial_index,
+        proof,
+        partial_merkle_tree.nodes.clone(),
```

Comment:
should use &partial_merkle_tree.nodes.clone()

Reply:
updated